ios-drag-drop.js

```diff
@@ -1,10 +1,11 @@
-(function(doc) {
+var module = (typeof module === 'object' && typeof module.exports === 'object') ? module : {};
```

Comment:
Is there value in defining a module variable?  If this module is running instead of a commonjs environment like browserify a module environment should already be defined, and if I'm reading this correctly this isn't necessary.